packages/backend/src/graphql/resolvers/incoming_payment.test.ts

```diff
@@ -68,4 +102,149 @@ describe('Incoming Payment Resolver', (): void => {
       }
     })
   })
+
+  describe('Mutation.createIncomingPayment', (): void => {
+    test.each`
+      description  | externalRef  | desc
+      ${'rent'}    | ${undefined} | ${'description'}
+      ${undefined} | ${'202201'}  | ${'externalRef'}
+    `('200 ($desc)', async ({ description, externalRef }): Promise<void> => {
+      const { id: paymentPointerId } = await createPaymentPointer(deps, {
+        asset
+      })
+      const payment = await createPayment({
+        paymentPointerId,
+        description,
+        externalRef
+      })
+
+      const createSpy = jest
+        .spyOn(incomingPaymentService, 'create')
+        .mockResolvedValueOnce(payment)
+
+      const input = {
+        paymentPointerId: payment.paymentPointerId,
+        incomingAmount: payment.incomingAmount,
+        expiresAt: payment.expiresAt
+      }
+
+      const query = await appContainer.apolloClient
+        .query({
+          query: gql`
+            mutation CreateIncomingPayment(
+              $input: CreateIncomingPaymentInput!
+            ) {
+              createIncomingPayment(input: $input) {
+                code
+                success
+                payment {
+                  id
+                  state
+                }
+              }
+            }
+          `,
+          variables: { input }
+        })
+        .then(
+          (query): IncomingPaymentResponse => query.data?.createIncomingPayment
+        )
+
+      expect(createSpy).toHaveBeenCalledWith(input)
+      expect(query.code).toBe('200')
+      expect(query.success).toBe(true)
+      expect(query.payment?.id).toBe(payment.id)
+      expect(query.payment?.state).toBe(SchemaPaymentState.Pending)
+    })
+
+    test('400', async (): Promise<void> => {
+      const createSpy = jest
+        .spyOn(incomingPaymentService, 'create')
+        .mockResolvedValueOnce(IncomingPaymentError.UnknownPaymentPointer)
+
+      const input = {
+        paymentPointerId: uuid(),
+        incomingAmount: {
+          value: BigInt(123),
+          assetCode: asset.code,
+          assetScale: asset.scale
+        },
+        expiresAt: new Date(Date.now() + 30_000)
```

Comment:
```suggestion
        paymentPointerId: uuid()
```
Ditto for `500` test below

Reply:
Removing the expiresAt field from the input causes the toHaveBeenCalledWith expectation to fail because of it's type as a date it doesn't just register as undefined.

Calling it with null instead also results in a date in 1970 popping up. Which from what I understand it should be substituting a date 90 days in the future if no date is supplied.

![image](https://user-images.githubusercontent.com/77615158/193807362-5eac62dd-2a16-444e-9abc-09bfb0adf08a.png)

![image](https://user-images.githubusercontent.com/77615158/193808884-52763189-5bb2-4287-a3a7-6659eec89068.png)

Reply:
This should fix it:
https://github.com/interledger/rafiki/pull/650/files/2d836218acdfc5a8050241e4bdece96f80b580da#r986830280

Reply:
Thank you.